src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/services/UnknownPatientService.kt

```diff
@@ -82,16 +82,15 @@ class UnknownPatientService(
 
   private fun UnknownPatient.dischargeToHospital(offenderNumber: String) =
     runCatching {
-      DischargeToHospitalRequest(
+      RestrictedPatient(
         offenderNumber,
-        "Historical hospital release added to NOMIS for addition to Restricted Patients",
         prisonCode,
         hospitalCode,
         prisonCode,
         hospitalOrderDate.atStartOfDay(),
-        noEventPropagation = true,
+        "Historical hospital release added to NOMIS for addition to Restricted Patients"
       )
-        .let { restrictedPatientsService.dischargeToHospital(it) }
+        .let { restrictedPatientsService.addRestrictedPatient(it, noEventPropagation = true) }
```

Comment:
By calling addRestrictedPatient we get the functionality we need - add to RP and call prison API to discharge - but we bypass the validation requiring a booking (which only applies to the UI, not for this one-off migration).